service/templates/search.html

```diff
@@ -19,12 +19,21 @@ <h1 class="heading-xlarge form-title">Find a title</h1>
                           <input type="text" class="form-control" name="search_term" id="search_term">
                       </div>
                       <div class="form-group">
-                          <input type="submit" value="Search" class="button">
+                          <input id="search_button" type="submit" value="Search" class="button">
                       </div>
                   </form>
                 </div>
             </div>
-
         </div>
     </main>
+    <script type="text/javascript">
+        function ga_title_search() {
+            search_term = document.getElementById('search_term').value
+            title_number_regex = '^([A-Z]{0,3}[1-9][0-9]{0,5}|[0-9]{1,6}[ZT])$'
+            if (search_term.match(title_number_regex)) {
+              ga('send', 'event', 'Search', 'TitleNumber', search_term);
+            }
+        }
+        document.getElementById('search_button').onclick = ga_title_search;
```

Comment:
How about keeping scripts in a separate js file?

I wonder if `ga_title_search` is always assigned to `onclick` event of `search_button`. Perhaps we should execute this line once we know the document has been loaded.